hsrb_impedance_control/launch/impedance_control_node.launch

```diff
@@ -5,8 +5,13 @@
   <node pkg="hsrb_impedance_control"
         name="impedance_control"
         type="hsrb_impedance_control_node">
-    <remap from="/wrench_state" to="$(arg namespace)/wrist_wrench/compensated"/>
-    <remap from="~configuration_names" to="~config_names"/>
+    <remap from="~wrench_state" to="$(arg namespace)/wrist_wrench/compensated"/>
+    <!-- 
+    The following remap is included from the original launch file (hsrb_impedance_control package).
+    It is, however, unclear what this should do and is therefore commented.
```

Comment:
Good that we add a comment for this but if we don't know what it does maybe its not smart to comment it